shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -70,6 +70,8 @@ object JsonLdUtils {
 
     const val EGM_OBSERVED_BY = "https://ontology.eglobalmark.com/egm#observedBy"
 
+    const val NGSILD_NAME_PROPERTY = "https://uri.etsi.org/ngsi-ld/name"
```

Comment:
this will no longer work when switching to core context v1.3 ... let's not add legacy code two weeks before the migration :)